util/env_windows_test_helper.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 The LevelDB Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#ifndef STORAGE_LEVELDB_UTIL_ENV_WINDOWS_TEST_HELPER_H_
+#define STORAGE_LEVELDB_UTIL_ENV_WINDOWS_TEST_HELPER_H_
+
+namespace leveldb {
+
+class EnvWindowsTest;
+
+// A helper for the Windows Env to facilitate testing.
+class EnvWindowsTestHelper {
```

Comment:
I am questioning if we really need these features on windows.  At least the mmap support should be added in a separate pull request after performance analysis.

Does windows have small open-file limits?  If not, the read-only-fd-limit can also be dropped, and this entire header file can go away.

Reply:
@ghemawat I believe there is no practical limit, I can remove this.